languages/python/oso/oso/oso.py

```diff
@@ -38,3 +38,24 @@ def is_allowed(self, actor, action, resource) -> bool:
             return True
         except StopIteration:
             return False
+
+    def get_allowed_actions(self, actor, resource, allow_unbound=False):
+        # Get allowed actions on the resource
+        results = self.query_rule("allow", actor, Variable("action"), resource)
+        actions = set()
+        for result in results:
+            action = result.get("bindings").get("action")
+            if type(action) == Variable:
+                # TODO: is this the correct behavior?
+                if not allow_unbound:
+                    raise exceptions.OsoError(
+                        """get_allowed_actions() found an unbound action. To get allowed actions, specify allowed actions in the policy.
+    E.g., instead of `allow(actor: User, _action, resource: MyResource);`,
+    use `allow(actor: User, action, resource: MyResource) if action in ["CREATE", "READ", "WRITE", "DELETE"];`
+                        """
+                    )
+                else:
+                    return ["UNBOUND"]
```

Comment:
Minor: probably best to avoid the term "unbound". Maybe `allow_wildcard`, and returning `"*"` for example might be a bit clearer?

Reply:
Yeah that's better.